api/app/Generators/NominationDocGenerator.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Generators;
+
+use App\Models\TalentNominationGroup;
+use App\Traits\Generator\GeneratesNominationDoc;
+use App\Traits\Generator\GeneratesUserDoc;
+
+class NominationDocGenerator extends DocGenerator implements FileGeneratorInterface
+{
+    use GeneratesNominationDoc, GeneratesUserDoc;
+
+    public function __construct(protected TalentNominationGroup $talentNominationGroup, public ?string $dir, protected ?string $lang)
+    {
+        $fileName = sprintf(
+            '%s - talent management nomination',
+            $this->sanitizeFileNameString($talentNominationGroup->nominee->first_name),
+        );
```

Comment:
This seems to be missing language localization, the filename needs to be different in French